CollectSamError/CollectSamErrorMetrics.wdl

```diff
@@ -51,7 +51,17 @@ task CollectSamErrorMetricsTask {
         ERROR_METRICS="ERROR:INSERT_LENGTH" \
         ERROR_METRICS="ERROR:CYCLE" \
         ERROR_METRICS="ERROR:GC_CONTENT" \
-        ERROR_METRICS="ERROR"
+        ERROR_METRICS="ERROR" \
```

Comment:
Can you remove the MAPPING_QUALITY stratifier from all stratifiers.  Add the -Minimum mapping quality 60 option.

Reply:
Also remove:
ERROR_METRICS="ERROR" \